cmdstanpy/model.py

```diff
@@ -942,17 +942,19 @@ def sample(
             num_threads = threads_per_chain
             one_process_per_chain = True
             assert isinstance(self.exe_file, str)  # make typechecker happy
-            info_dict = self.exe_info()
+            info_dict = self.exe_info
             stan_threads = info_dict.get('STAN_THREADS', 'false').lower()
+            # run multi-chain sampler if possible, unless algo is fixed_param
             if (
                 force_one_process_per_chain is None
                 and not cmdstan_version_before(2, 28, info_dict)
+                and not fixed_param
```

Comment:
Should probably just use `force_one_process_per_chain = force_one_process_per_chain or fixed_param`